pages/programs/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { ProgramsList } from '../../lib/components/programs-list';
+import { VStack, Text, Container } from '@chakra-ui/react';
+import OSLHeader from '../../lib/components/header';
+
+const Programs: React.FC = () => {
+  return (
+    <Container maxW={'container.lg'} mb="100px" fontSize="24px" paddingTop="24px">
+      <VStack align="flex-start">
+        <div data-cy="programs-heading">
+          <OSLHeader
+            header="June Online Events"
+            style={{ width: '100%', fontWeight: 'bolder', marginBottom: '30px' }}></OSLHeader>
```

Comment:
Prefer Box to style

Reply:
@rymaju what do you mean by Box? in our Header component `/components/header/index.tsx`, we use a Box and pass in a style attribute already